app/models/manageiq/providers/ibm_cloud/inventory/parser/power_virtual_servers.rb

```diff
@@ -123,14 +125,15 @@ def images
   def volumes
     collector.volumes.each do |vol|
       persister.cloud_volumes.build(
-        :ems_ref       => vol['volumeID'],
-        :name          => vol['name'],
-        :status        => vol['state'],
-        :bootable      => vol['bootable'],
-        :creation_time => vol['creationDate'],
-        :description   => 'IBM Cloud Block-Storage Volume',
-        :volume_type   => vol['diskType'],
-        :size          => vol['size']&.gigabytes
+        :availability_zone => persister.availability_zones.lazy_find(persister.cloud_manager.name),
```

Comment:
Just need to update the lazy_find to match now

Reply:
Does lazy_key off of uid_ems only?